Client/mods/deathmatch/logic/CPacketHandler.h

```diff
@@ -38,6 +38,7 @@ class CPacketHandler
         ELEMENT_FAILURE,
         GENERAL_REFUSED,
         SERIAL_VERIFICATION,
+        SERIAL_DUPLICATE,
```

Comment:
Move SERIAL_DUPLICATE to the end of the list. It breaks packets from an old sever

Reply:
Thanks!